packages/vats/test/test-lib-chainStorage.js

```diff
@@ -8,14 +8,14 @@ test('makeChainStorageRoot', async t => {
   const rootPath = 'root';
   const { rootNode, messages } = makeFakeStorageKit(rootPath);
   t.deepEqual(
-    rootNode.getStoreKey(),
+    await rootNode.getStoreKey(),
```

Comment:
This `await` should probably move up out of function arguments.

Reply:
Okay, will do that.